### PR TITLE
Release 1.10.6

### PR DESCRIPTION
**Pull request by mrclay**

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
Wrong version number.

**Comment by juho-jaakkola** [comment]
Wait, no it isn't. 1.10.5 is just missing from the community release list: https://elgg.org/previous.php

**Comment by mrclay** [comment]
I'm going to manually add the missing ones

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+* Mariano Aguero (1)
+* akudan (1)
+* Steve Clay (1)
+* Jeroen Dalsem (1)
+
+#### Bug Fixes
+
+* **views:** Sticky values now get passed into views extending register/extend ([e241e82e](https://github.com/Elgg/Elgg/commit/e241e82eef3ac57e8cffdfdad164fe49372ddfd7), closes [#8873](https://github.com/Elgg/Elgg/issues/8873))
+* **memcache:** don't store a copy of $CONFIG in file objects ([beb90891](https://github.com/Elgg/Elgg/commit/beb9089129a0a06b36200f3f8d214c7ed8f94f42)), closes [#9081](https://github.com/Elgg/Elgg/issues/9081))
+* **messageboard:** provide correct link to users messageboard (Fixes [#8170](https://github.com/Elgg/Elgg/issues/8170))
```
Comment (mrclay) [inline CHANGELOG.md]: I was too lazy to link the commits for all these

**Comment by mrclay** [comment]
Manually updated Changelog

**Comment by iionly** [comment]
I see quite a number of language files with upper case characters in file name added. Shouldn't these filenames use only lower case letters? Is it a release script issue as I thought this would be fixed in later versions of Elgg at least.

**Comment by mrclay** [comment]
Crap. The release *was* fixed, but I have to backport the fix to this branch.

**Comment by mrclay** [comment]
Backporting the transifex process fixes to this branch looks really hard. I might just manually remove the unwanted files this time. I want to get this out.

**Comment by mrclay** [comment]
This won't contain language updates. Since most of them were for languages that would be filtered out in fixed branches I just dropped them all.

Thinking more about it it's not obvious that updating translations in bugfix branches is a good idea.

**Pull request merged**